content/wiki/canonical_lints.md

```diff
@@ -9,7 +9,14 @@ All Linebender projects should include the following set of lints in their `Carg
 # This one may vary depending on the project.
 rust.unsafe_code = "forbid"
 
+# These two should be added to projects with an MSRV of 1.81 or higher
```

Comment:
Why special case these? The lints themselves already check the msrv to see if they should run

Reply:
Because actually addressing these lints requires features (expect annotations and allow reasons) that came out in 1.81.

Reply:
I'm not sure I follow. These lints only trigger if your package's MSRV is above 1.81 anyway, so why does it matter if you need features from Rust 1.81 to address them? If you're getting this error, surely you have those features?

Reply:
So the lints are basically no-ops if `rust-version = "1.80"`, even if the user has a Rust 1.81 compiler?

Reply:
Yeah.